includes/settings/js/src/components/DeleteModelModal.jsx

```diff
@@ -6,6 +7,7 @@ import { removeSidebarMenuItem } from "../utils";
 import { getRelationships } from "../queries";
 import { showError } from "../toasts";
 import { useHistory } from "react-router-dom";
+import { sendEvent } from "../../../../shared-assets/js/analytics";
```

Comment:
Worth using the alias here for consistency?

```suggestion
import { sendEvent } from "acm-analytics";
```

Reply:
Ah yes, I think my editor auto-added the path. Good catch.